modules/ROOT/pages/_partials/partner-and-customer-guide.adoc

```diff
@@ -50,6 +50,8 @@ Mule 4.6.x exposes Java EE libraries differently than earlier Mule versions. You
     </dependencies>
   </dependencyManagement>
 ----
++
+For more information, see xref:mule-sdk::java-ee-libraries.adoc[].
```

Comment:
same as with the other PR. Does it make sense for application migration guide to share the same content as the documentation on how to actually develop new extensions? Different point of views and level of details iMO

Reply:
@marianogonzalez Does the existing information in migration guide look fine? In that case, should I just remove these "For more information," reference links?

Reply:
I don't think so, the feedback is quite the opposite actually.  The SDK documentation should explain what needs to happen on the connector and how those dependencies should be handled. This one should explain what to do at the app level. Completely different POVs and explanations.  These should be differnet pieces of content all together

Reply:
In this migration guide, just looking at the "Release Your Custom Connector" section, the info in that section has the steps needed to add the @JavaVersionSupport annotation. What exactly do I need to add or remove from that section? That section has the relevant info for a user to mark their connector as J17 compatible and gives some context as well.